### DISCUSSION TITLE
Set a default sort column option for a collection

### DISCUSSION THREAD

**Discussion by ollebergkvist** [payloadcms/payload#921] [Feature Requests & Ideas]
Currently, Payload supports a defaultColumns property, would be great if we could get an additional prop, perhaps:
sortData where you can define a default column to sort by and also set if it should be in ASC or DESC order.

**Comment by jmikrut** [comment]
Hey @ollebergkvist — this is a great request!

We would happily accept a PR for this functionality, but otherwise I'm sure we can get to it at some point. Looking at how the default columns are done is a great starting point for this PR.

Thanks for the idea!

**Reply by ollebergkvist** [reply]
Cool! I believe I can try and have a go at this next weekend.
